createdisk.sh

```diff
@@ -78,9 +78,9 @@ EOF
     # also in case of microshift the ports like 2222, 443, 80 ..etc need to be manually added
     # and OCP/OKD/podman bundles have it disabled by default.
     ${SSH} core@${VM_IP} -- sudo systemctl disable firewalld
-    # Copy the sample microshift config and update the base domain with crc base domain
-    ${SSH} core@${VM_IP} -- sudo cp /etc/microshift/config.yaml.default /etc/microshift/config.yaml
-    ${SSH} core@${VM_IP} -- "sudo sed -i 's/#baseDomain: .*/baseDomain: ${SNC_PRODUCT_NAME}.${BASE_DOMAIN}/g' /etc/microshift/config.yaml"
+    ${SSH} core@${VM_IP} -- "sudo podman run --rm  -v /etc/microshift:/workdir:Z docker.io/mikefarah/yq:4.35.2 eval '.dns.baseDomain = \"${SNC_PRODUCT_NAME}.${BASE_DOMAIN}\"'  config.yaml.default | sudo tee /etc/microshift/config.yaml"
+    # Make sure `baseDomain` is set to crc.testing
+    ${SSH} core@${VM_IP} -- "grep 'baseDomain: ${SNC_PRODUCT_NAME}.${BASE_DOMAIN}' /etc/microshift/config.yaml"
```

Comment:
This will match `#baseDomain: ...`

Reply:
@cfergeau how will this match `#baseDomain: example.com` doesn't it match only `baseDomain: crc.testing` or `#baseDomain: crc.testing` (which is not possible)?

Reply:
yes but `#baseDomain: crc.testing` why this will even there in this file the commented one is `#baseDomain: example.com`?

Reply:
> yes but #baseDomain: crc.testing why this will even there in this file,

I don't know how it would get there, but the aim of this check is to catch future regressions around domain name setting. If we can avoid false negatives, even if they can't happen now, it's better :)

Reply:
Again not want to block this PR on something might happen in future. Added the check false negative.